test/xds/xds_server_rbac_test.go

```diff
@@ -605,6 +630,12 @@ func (s) TestRBACHTTPFilter(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			func() {
```

Comment:
This is not new but do you know why such a anonymous func is needed here?

Reply:
No clue, I'm just working into the existing test